recipe-server/client/control_new/state/serviceInfo/selectors.js

```diff
@@ -2,8 +2,8 @@ export function getCurrentUser(state, defaultsTo = null) {
   return state.app.serviceInfo.get('user', defaultsTo);
 }
 
-export function isPeerApprovalEnforced(state, defaultsTo = true) {
-  return state.app.serviceInfo.get('peer_approval_enforced', defaultsTo);
+export function isPeerApprovalEnforced(state) {
+  return state.app.serviceInfo.get('peer_approval_enforced') !== false;
```

Comment:
ensures that `peer_approval_enforced` is always `true` unless explicitly set to `false`.